### PR TITLE
Monica's Blog Post

### PR DESCRIPTION
**Pull request by monicayao**
I was debating if I should make a PR of the post or wait for further instructions, but since Ruslan did, I thought I should too! 😸

### CODE REVIEW

File: _posts/blog/2019-05-21-Monica's-Introduction [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+share: true
+author: monicayao
+---
+
+## About Me
+
+My name is Monica and I am currently an undergraduate student at Harvey Mudd College (located in the beautiful Southern California). 
+I am joining the Whitaker Lab as a part of the Google Summer of Code program. My interests for computer science are very recent - 
+a year ago, I had wanted to be a surgeon! I have discovered so many useful computer science applications for the medical field. 
+Through working with the Tedana library, I hope to assist in denoising fMRI data, and dive deeper into the real life applications of CS!
```
Comment (KirstieJane) [inline _posts/blog/2019-05-21-Monica's-Introduction]: I think I’ve always written tedana in all lowercase, like the package name. What do you think?

**Review by KirstieJane — COMMENTED** [review]
This is really fantastic @monicayao!! Thank you for sharing with me!

I know this is really annoying, but would you mind restructuring the markdown to have each sentence on a new line. I think it’s easier to review complete sentences and track version control across whole sentences.

Thank you!

File: _posts/blog/2019-05-21-Monica's-Introduction [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<figure class="half">
+  <img src="/images/Monica_Introduction/food1.jpg">
+  <img src="/images/Monica_Introduction/food2.jpg">
+</figure>
+<figure class="half">
+  <img src="/images/Monica_Introduction/food3.jpg">
+  <img src="/images/Monica_Introduction/food4.jpg">
+  <figcaption> Here are some of the recent food pictures I took! </figcaption>
+</figure>
+
```
Comment (KirstieJane) [inline _posts/blog/2019-05-21-Monica's-Introduction]: Love these!! 😋😋😋

File: _posts/blog/2019-05-21-Monica's-Introduction [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+### Why Tedana and what do I want to achieve?
+I am excited about helping Tedana because it is a very small, personal project. The science and process behind it is very specific (and difficult to understand).
+The reason why I chose Tedana is because I have always been practicing Test-Driven Development whether in school or in my own projects, and utilizing the skills in 
+the real world, on an actual library, is something I wanted to experience. 
+
+I have taken multiple machine learning classes, and reading about the progress in machine learning in medical imaging piqued my interests in the other steps of the pipeline. 
+How does one make sure that the data fed into machine learning is good, with no noise? 
+
+That is exactly what Tedana does! Tedana denoises the data from multi-echo fMRI, and produces clean data that is useful for statistical analysis.
```
Comment (KirstieJane) [inline _posts/blog/2019-05-21-Monica's-Introduction]: This is a great description of tedana! Love it!

**Review by KirstieJane — COMMENTED** [review]
Whoops, submitted my review too early 🤦🏼‍♀️

**Comment by monicayao** [comment]
@KirstieJane Finished the edits! 😸

File: _posts/blog/2019-05-21-Monica's-Introduction.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+author: monicayao
+---
+
+## About Me
+
+My name is Monica and I am currently an undergraduate student at Harvey Mudd College (located in the beautiful Southern California). 
+I am joining the Whitaker Lab as a part of the Google Summer of Code program. 
+My interests for computer science are very recent - a year ago, I had wanted to be a surgeon! 
+I have discovered so many useful computer science applications for the medical field. 
+Through working with the ```tedana``` library, I hope to assist in denoising fMRI data, and dive deeper into the real life applications of CS!
```
Comment (KirstieJane) [inline _posts/blog/2019-05-21-Monica's-Introduction.md]:
```suggestion
Through working with the [`tedana`](https://tedana.readthedocs.io/en/latest/) library, I hope to assist in denoising fMRI data, and dive deeper into the real life applications of CS!
```

**Review by KirstieJane — APPROVED** [review]
Looks awesome @monicayao!! I'm sorry for the slow review.

I've added one suggestion. You should be able to hit the green button and merge when you've accepted it :sparkles:

**Pull request merged**